frappe/email/doctype/email_account/email_account.json

```diff
@@ -18,6 +18,10 @@
   "awaiting_password",
   "ascii_encode_password",
   "column_break_10",
+  "use_google_oauth",
+  "authorize_google_api_access",
+  "google_refresh_token",
+  "google_access_token",
```

Comment:
Can't these be general-purpose? What if we want to enable OAuth for another email service... 4 new fields?
```suggestion
  "use_oauth",
  "authorize_api_access",
  "refresh_token",
  "access_token",
```

Reply:
hmm it can be done but then we need to have the authentication url as well which redirects us to the service page and the user allows/permits us to use their info.

kept it simple (only google) for the time being due to that (and the date for google stopping less secure apps is today so 🤷‍♂️ )

Reply:
we could use the generic naming but only enable it for gmail for the time being as well (?)